amenities/index.html

```diff
@@ -8,8 +8,7 @@
 <link rel="icon" href="/favicon.png" type="image/png" />
 <script type="text/javascript" src="../js/css.min.js"></script>
```

Comment:
If you've removed async load css feature, make sure to remove css.min.js as well.